src/nav/root.yml

```diff
@@ -4,8 +4,12 @@ pages:
   - title: Start here
   - title: Start using New Relic
     path: /docs/new-relic-solutions/get-started/intro-new-relic
+  - title: My application is slow
```

Comment:
Why do we have two entries for this category in the sidebar?
<img width="264" alt="image" src="https://user-images.githubusercontent.com/55203603/236495354-718ce4b9-9516-4078-a51e-c6f79b2fe785.png">